### ISSUE TITLE
PLG_SYSTEM_ENVIRONMENTDETECT_CANT_DETECT_SYSTEM

### ISSUE THREAD

**Issue by pe7er** [coolcat-creations/ccc-mountain#12]
At https://browsercache-legen.nl/ on the English language I click on "Detect my Browser", then on my Linux machine with Chrome/Firefox appears the following message:
"Sorry we have no descriptions for your operating system"

When I use the **Dutch** or **French language**, the language string is **not translated** and
PLG_SYSTEM_ENVIRONMENTDETECT_CANT_DETECT_SYSTEM
appears instead.

**Comment by zero-24** [comment]
The message comes from my plugin: https://github.com/zero-24/plg_system_environmentdetect

Can you send the correct translation there? This should than make them available with the next update :)

Usually I would expect the en-GB file to be loaded when there is no dedicated file for the choosen language maybe @coolcat-creations can take a look and share the language debug console which file joomla tries to load?

> "Sorry we have no descriptions for your operating system"

This also seams to be not the original message? Maybe an intended override? I'm happy to apply such changes to the original plugin for sure :)

**Comment by pe7er** [comment]
@coolcat-creations Could you add your Dutch language file to https://github.com/zero-24/plg_system_environmentdetect/tree/master/language ?

@zero-24 I would translate PLG_SYSTEM_ENVIRONMENTDETECT_CANT_DETECT_SYSTEM to:
"Sorry, er is geen beschrijvingen beschikbaar voor uw besturingssysteem"

**Comment by coolcat-creations** [comment]
Thank you, I will do.

**Comment by coolcat-creations** [comment]
I added the language files to the original Plugin repo, but I can't fnd why the plugin does not load the language files. Maybe it has to do with the multidomain plugin? Debugging does not work at all.

**Comment by zero-24** [comment]
> Debugging does not work at all.

Debugging does not work at all?

**Comment by coolcat-creations** [comment]
No, maybe because of the language filter plugin. I don't see any debugging console even if turned on.

**Comment by zero-24** [comment]
Please check: https://github.com/zero-24/plg_system_environmentdetect/releases/tag/1.0.3 This adds the translations and also moves the ini to sys.ini files.
